src/components/ui/accordion.tsx

```
@@ -0,0 +1,58 @@
+"use client"
```

ESLint

[error] Replace `"use·client"` with `'use·client';`

(prettier/prettier)